packages/ListBox/test/cypress/ListBox.a11y.cypress.js

```diff
@@ -0,0 +1,93 @@
+import { getStoryUrlPrefix } from "../../../../.storybook/storyTree";
+import { toggleDropdown } from "../helpers/toggleHelpers";
+import selectors from "../helpers/selectors";
+
+const storyPrefix = `${getStoryUrlPrefix("ListBox")}`;
+
+function arrowUpKeyPress() {
+  cy.focused().type("{uparrow}", { force: true });
+}
+
+function arrowDownKeyPress() {
+  cy.focused().type("{downarrow}", { force: true });
+}
+
+function optionHasFocus(optionName) {
+  cy.get(selectors.popoverList)
+    .findByText(optionName)
+    .should("have.focus");
+}
+
+describe("ListBox MultiSelect a11y", () => {
+  beforeEach(() => {
+    cy.visitStorybook(`${storyPrefix}-backyard-tests--a-11-y-tests-story`);
+    toggleDropdown();
+    cy.wait(350);
+  });
+
+  it("focus should be moved to the filter input once dropdown is opened", () => {
+    cy.get(selectors.filterInput).should("have.focus");
```

Comment:
Maybe this could actually go in the `beforeEach` to make sure the component is ready to be interacted with?